README.md

```diff
@@ -91,12 +91,13 @@ or
 
 Once the build is completed, you will find the resulting output i.e. images, dtbs and rootfs at "bin/pistachio".
 
-Example output for V0.9.4:
+Example output for V0.10.x:
```

Comment:
Need to update following things in readme:
1.we need to use usbboot or mmcboot for first time since its  not possible to sysupgrade the 10.4 image with this new openwrt-0.10.x-pistachio-marduk-marduk-squashfs-factory.ubi
2.we need to copy openwrt-0.10.x-pistachio-marduk-marduk-kernel.itb as fitImage in the rootfs on USB stick.
3.then use `env default -a` to wipe out uboot env variables.
4. then only run mmacboot or run usbboot will work to boot fitImage supported openwrt builds.
5. then you can use sysupgrade -n /tmp/openwrt-0.10.x-pistachio-marduk-marduk-squashfs-factory.ubi so flash the image into NAND.